<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckDialog</class>
 <widget class="QDialog" name="CheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人工复核</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>原始评论：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="cmtTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>投票对象，留空则无:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="voteLineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="confirmPushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
